context/Web3Context/Web3Context.tsx

```diff
@@ -150,25 +184,94 @@ const Web3Store: React.FC<Web3ContextProps> = ({
+        const calculatorEl = document.querySelector('div[class*="Calculator"]') as HTMLElement;
+        const marginModalEl = document.querySelector('div[class*="AccountModal"]') as HTMLElement;
+        if(calculatorEl){
+            calculatorEl.removeAttribute('style');
+        }
+        if (marginModalEl) {
+            marginModalEl.removeAttribute('style');
+        }
+
+        setTutorialComplete();
+    };
+
+    const highlightDots = (e: HTMLDivElement) => {
+        e.addEventListener('click', function () {
+            const navDots: Array<any> = Array.from(document.querySelectorAll('nav[data-tour-elem="navigation"] button'));
+            var currentIndex = 0;
+            // Wait for Reactour to apply styling
+            setTimeout(function(){
+                navDots.map((dot, i) => { 
+                    if(dot.classList.contains('reactour__dot--is-active')){
+                        currentIndex = i;
+                    }
+                });
+                navDots.slice(0, currentIndex).map((dot) => {
+                    dot.classList.add('reactour__dot--is-active');
+                });
+            }, 10);
+        });
+        // Also prevent body scrolling when tour open
+        disableBodyScroll(e);
+    }
+
     return (
-        <Web3Context.Provider
-            value={{
-                account: account,
-                network: network,
-                ethBalance: ethBalance,
-                web3: web3,
-                wallet: wallet,
-                onboard: onboard,
-                isReady: isReady,
-                checkIsReady,
-                resetOnboard,
-                handleConnect,
-                config,
-                isMobile: !!onboardState?.mobileDevice,
-            }}
-        >
-            {children}
-        </Web3Context.Provider>
+        <>
+            <Web3Context.Provider
+                value={{
+                    account: account,
+                    network: network,
+                    ethBalance: ethBalance,
+                    web3: web3,
+                    wallet: wallet,
+                    onboard: onboard,
+                    isReady: isReady,
+                    checkIsReady,
+                    resetOnboard,
+                    handleConnect,
+                    config,
+                    isMobile: !!onboardState?.mobileDevice,
+                }}
+            >
+                {children}
+            </Web3Context.Provider>
+            <Tour
+                onRequestClose={closeTour}
+                steps={tourConfig as Array<any>}
+                maskSpace={0}
+                isOpen={isTourOpen}
+                maskClassName="mask"
+                className="helper"
+                rounded={5}
+                showNumber={false}
+                updateDelay={0}
+                onAfterOpen={(e) => highlightDots(e)}
+                onBeforeClose={(e) => enableBodyScroll(e)}
+            />
```

Comment:
Since this will render on insurance page. But we dont have the insurance page tutorial yet